mate-base/mate-settings-daemon/files/mate-settings-daemon-1.26.0-fix-buid-with-clang16.patch

```diff
@@ -0,0 +1,14 @@
+Bug: https://bugs.gentoo.org/881315
+Author: David Carlos Manuelda <StormByte@gmail.com>
+--- a/plugins/a11y-keyboard/msd-a11y-keyboard-atspi.c
++++ b/plugins/a11y-keyboard/msd-a11y-keyboard-atspi.c
+@@ -128,7 +128,7 @@
+         /* init AT-SPI if needed */
+         atspi_init ();
+ 
+-        self->listener = atspi_device_listener_new (on_key_press_event,
++        self->listener = atspi_device_listener_new ((AtspiDeviceListenerCB)on_key_press_event,
```

Comment:
I'm not yet sure if this is actually right. I need to check.

Reply:
My initial patch was to remove const from `on_key_press_event`'s paramer
```diff
 static gboolean
-on_key_press_event (const AtspiDeviceEvent *event,
+on_key_press_event (AtspiDeviceEvent *event,
                     void                   *user_data G_GNUC_UNUSED)
 {
         /* don't ring on capslock itself, that's taken care of by togglekeys
```

Reply:
Thanks for asking - I think the patch as it is (the cast) is better now I looked at it harder.